### ISSUE TITLE
Custom Monitor definition did not create terraform

### ISSUE THREAD

**Issue by mjhuber** [reactiveops/rodd#24]
I created a custom definition like below.  When i run the `pentagon add datadog.monitors` command it is ignored and the terraform was not created.

```yaml
-  name: "[${environment}] Node is not Ready"
    tags: ['${environment}']
    include_tags: false
    no_data_timeframe: 2
    silenced: {}
    new_host_delay: 900
    notify_no_data: false
    renotify_interval: 0
    query: kubernetes_state.node.ready.over(kubernetescluster:${cluster}).by(host).last(4).count_by_status()
    message: |
      {{#is_warning}}
      A Node is not ready!
      Cluster: {{kubernetescluster.name}}
      Host: {{host.name}}
      IP: {{host.ip}}
      {{check_message}}
      {{/is_warning}}
      {{#is_alert}}
      A Node is not ready!
      Cluster: {{kubernetescluster.name}}
      Host: {{host.name}}
      IP: {{host.ip}}
      {{check_message}}
      {{/is_alert}}
      {{#is_recovery}}
      Node is now ready.
      Cluster: {{kubernetescluster.name}}
      Host: {{host.name}}
      IP: {{host.ip}}
      {{/is_recovery}}
    type: service check
    timeout_h: 0
    thresholds: {critical: 3, warning: 2, ok: 2}
```

**Comment by sudermanjr** [comment]
Did you try running pentagon with `--log-level=DEBUG` ?  There may be more info there.

**Comment by mjhuber** [comment]
closing, this was fixed with #29
